float64 pred_time
float64 pred_height
float64 pred_traj_height
float64 dv_dth_alpha
